### PR TITLE
Add util method for assigning groups to a user in a single call

### PR DESCRIPTION
**Pull request by bdemers**
Adds UserBuilder.addGroup

Fixes: #112

### CODE REVIEW

File: api/src/main/java/com/okta/sdk/resource/user/UserBuilder.java [hunk 1]
```diff
@@ -18,7 +18,9 @@
 import com.okta.sdk.client.Client;
 import com.okta.sdk.lang.Classes;
 
+import java.util.List;
```
Comment (bretterer) [inline api/src/main/java/com/okta/sdk/resource/user/UserBuilder.java]: Is `java.util.List` used anywhere in the `UserBuilder`?

Comment (bdemers) [reply api/src/main/java/com/okta/sdk/resource/user/UserBuilder.java]: Nope, that should have failed CI... I fix that and figure out why that didn't get picked up by the static analysis

**Comment by bdemers** [comment]
damn, i also misspelled the 'Fixes' in the footer...

**Comment by bdemers** [comment]
NOTE: force pushed to update initial commit footer

File: pom.xml [hunk 2]
```diff
@@ -220,6 +220,27 @@
...
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-pmd-plugin</artifactId>
+                    <configuration>
+                        <excludeRoots>
+                            <!-- exclude the generated code from PMD, Findbugs will still run -->
+                            <excludeRoot>${project.build.directory}/generated-sources/swagger-codegen</excludeRoot>
+                        </excludeRoots>
```
Comment (bdemers) [inline pom.xml]: Previous config was inherited, actual change removes these lines:
```xml
 <!-- generated resources -->
                            <exclude>**/com/okta/sdk/resource/**/*.java</exclude>
                            <exclude>**/com/okta/sdk/client/Client.java</exclude>
<exclude>**/com/okta/sdk/impl/resource/Default*.java</exclude>
<exclude>**/com/okta/sdk/impl/resource/**/Default*.java</exclude>
<exclude>**/com/okta/sdk/impl/client/DefaultClient.java</exclude>
```
And replaces them with the `excludeRoots` config

**Pull request merged**